core/cas-server-core-events-api/src/main/java/org/apereo/cas/support/events/listener/CasAuthenticationAuthenticationEventListener.java

```diff
@@ -42,14 +42,12 @@ public class CasAuthenticationAuthenticationEventListener implements CasAuthenti
 
     private final GeoLocationService geoLocationService;
 
-    private CasEvent prepareCasEvent(final AbstractCasEvent event) {
+    private CasEvent prepareCasEvent(final AbstractCasEvent event,final ClientInfo clientInfo) {
```

Comment:
Please run checkstyle on your changeset.

This should be, i.e.,

```
private CasEvent prepareCasEvent(final AbstractCasEvent event, final ClientInfo clientInfo) {
```

Reply:
This file should have the space now.  I'll check the rest when I make changes for the other comments.